jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenterTest.java

```diff
@@ -58,6 +62,7 @@
 import org.uberfire.mvp.Commands;
 import org.uberfire.mvp.PlaceRequest;
 
+import static java.util.Collections.*;
 import static org.jbpm.workbench.es.client.editors.util.JobUtils.createRequestSummary;
 import static org.jbpm.workbench.es.model.ExecutionErrorDataSetConstants.EXECUTION_ERROR_LIST_DATASET;
```

Comment:
Not related with the PR changes but this EXECUTION_ERROR_LIST_DATASET could be removed :)

Reply:
+1 somehow my IntelliJ is not picking up all of these unused imports :/